[nativert] Address tooling setup for torch/nativert/

zhxchen17 opened the pull request:
Summary:
As discussed with @malfet , we're porting nativert code to torch/nativert/.
Following up some concerns over the new directory, I'm trying to setup the tooling on OSS so various things (like linters) can run on torch/nativert/ properly.

Test Plan: CI

Differential Revision: D74407808

Skylion007 commented on .lintrunner.toml:
```diff
@@ -232,6 +234,10 @@ include_patterns = [
     'torch/csrc/**/*.cpp',
     'torch/csrc/jit/serialization/*.h',
     'torch/csrc/jit/serialization/*.cpp',
+    'torch/nativert/*.h',
```
You need to also modify the `.clang-tidy` file and the header include regex in that file

zhxchen17 replied:
Checked .clang-tidy and seems we already include the entire `torch/` there. https://github.com/pytorch/pytorch/blob/main/.clang-tidy

zhxchen17 committed:
[nativert] Address tooling setup for torch/nativert/ (#153164)

Summary:
Pull Request resolved: https://github.com/pytorch/pytorch/pull/153164

As discussed with malfet, we're porting nativert code to torch/nativert/.
Following up some concerns over the new directory, I'm trying to setup the tooling on OSS so various things (like linters) can run on torch/nativert/ properly.

Test Plan: CI

Reviewed By: dolpm

Differential Revision: D74407808
